trac-plugins/aws-config/build.gradle

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 Accenture Global Solutions Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id "java-library"
+    id "idea"
+}
+
+apply from: "${rootProject.projectDir.path}/gradle/base-java.gradle"
+
+
+dependencies {
+
+    api project(':trac-lib-common')
+
+    // Apache Commons Components
+    implementation group: 'commons-cli', name: 'commons-cli', version: "$commons_cli_version"
+
+    implementation group: 'com.amazonaws', name: 'aws-java-sdk', version: "1.11.163"
```

Comment:
Please could you add aws_sdk_version into config.gradle with the other version settings